lasagne/layers/corrmm.py

```diff
@@ -18,7 +18,7 @@
 ]
 
 
-if not theano.config.device.startswith("gpu"):
+if not theano.config.device.startswith("gpu"):  # pragma: no cover
```

Comment:
I'm not sure if this is the best we can do here - but it's hard to test this without spoofing. Might be worth it though, people could try to use these layers without a compatible GPU and we should make sure that they get an understandable error message in this case. Any ideas?